components/atoms/a-picture.vue

```diff
@@ -0,0 +1,75 @@
+<template>
```

Comment:
I think we don't need this component anymore 🤔 
We can pass to SfImage transformed data based on `'supports-webp'`.